vumi/persist/fake_redis.py

```diff
@@ -513,6 +514,21 @@ def persist(self, key):
             return 1
         return 0
 
+    # HyperLogLog operations
+
+    @maybe_async
+    def pfadd(self, key, *values):
+        hll = self._setdefault_key(key, HyperLogLog(0.01))
```

Comment:
Both Redis and `HyperLogLog(0.01)` use 14 bits from the hash function, which gives an error of approximately 0.81%. `HyperLogLog(0.0081)` is just over the boundary between 14 and 15 bits, so it actually ends up with a different (smaller) error. (`0.0082` uses 14 bits, but it's uglier than `0.01`.)